packages/components/molecules/f-media-element/src/components/_tests/MediaElement.test.js

```diff
@@ -56,75 +56,6 @@ describe('MediaElement.vue', () => {
             expect(textWrapper.text()).toEqual(mockText);
         });
 
-        it('should apply stack class when stacked prop is true', () => {
```

Comment:
Should we have new tests for the flex class going in?

Reply:
Yep have added tests to support the new prop